### ISSUE TITLE
Feature: Adding support for Office 365.

### ISSUE THREAD

**Issue by Shreypatel13ll** [openchatai/OpenChat#52]
### Description
I would like to request the addition of support for Office 365 in OpenChat.

### Proposed Solution
- PowerPoint, Word, OneNote: By processing raw text and specifying different weights for headings, descriptions, etc.

- Excel: By processing the rows and columns as tables, OpenChat can interpret the data and provide relevant information. If any prompts or formulas are included in the Excel file, OpenChat can consider them during the analysis. In cases where external prompts are required, users can be prompted to provide additional information.

**Comment by gharbat** [comment]
It's indeed on the roadmap to support Office 360, will keep you updated!

**Comment by Shreypatel13ll** [comment]
Should I start working on it, or is it assigned to someone else.

**Comment by gharbat** [comment]
Yes you can! Please let me know if you need any help or support.

**Comment by Shreypatel13ll** [comment]
sure !!
